### PR TITLE
Installation Guide

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

File: public/scripts/installation.sh [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    sudo pacman -Syyy curl git
+}
+
+install_deps_ubuntu () {
+    sudo apt-get install -y curl git
+}
+
+install_deps_ubuntu () {
+    sudo apt-get install -y curl git
+}
```
Comment (ruilopesm) [inline public/scripts/installation.sh]: This function is repeated twice.

File: src/pages/docs/installation.md [hunk 2]
```diff
@@ -1,71 +1,88 @@
...
+
+| OS | Command |
+| Arch Linux | sudo pacman -S docker docker-compose |
+| Fedora | [Refer to this guide](https://docs.docker.com/engine/install/fedora) |
+| Ubuntu | [Refer to this guide](https://docs.docker.com/engine/install/ubuntu) |
+| macOS | brew install docker && brew install docker-compose|
+
+## Optional Software
+
+### Web Browsers
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]: Please refer [CORS Everywhere](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/)

File: public/scripts/installation.sh [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+# https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies
+install_asdf () {
+    git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.11.2
+
+    printf ". \"$HOME/.asdf/asdf.sh\"\n" >> ~/.bashrc
+    printf ". \"$HOME/.asdf/completions/asdf.bash\"\n" >> ~/.bashrc
+
+    # Apply the changes so that ASDF in in $PATH
```
Comment (ruilopesm) [inline public/scripts/installation.sh]:
```suggestion
    # Apply the changes so that ASDF is in $PATH
```

File: src/pages/docs/installation.md [hunk 4]
```diff
@@ -1,71 +1,88 @@
...
+
+You can download Postman using the following commands:
+
+
+| OS | Command |
+|-|-|
+Arch based | yay -S postman-bin
+Ubuntu / Debian based | sudo snap install postman
+Fedora / Red Hat | [Refer to this tutorial](https://mamchenkov.net/wordpress/2020/01/30/install-postman-on-fedora-31/)
+
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
macOS | brew install --cask postman
```

File: public/scripts/installation.sh [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+# https://docs.docker.com/engine/install/ubuntu/
+install_docker_ubuntu() {
+    sudo apt-get remove docker docker-engine docker.io containerd runc
+    sudo apt-get update
+
+    sudo apt-get install -y ca-certificates curl gnupg lsb-release
+    sudo mkdir -m 0755 -p /etc/apt/keyrings
+
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+    sudo mkdir -m 0755 -p /etc/apt/keyrings
```
Comment (ruilopesm) [inline public/scripts/installation.sh]: This line of code is repeated twice.

File: public/scripts/installation.sh [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+install_deps_ubuntu () {
+    sudo apt-get install -y curl git
+}
+
+install_deps_fedora() {
+    sudo dnf -y install curl git
+}
+
+docker_no_sudo() {
+    # Run without sudo
```
Comment (ruilopesm) [inline public/scripts/installation.sh]:
```suggestion
    # In order to run docker command without sudo
```

File: src/pages/docs/installation.md [hunk 7]
```diff
@@ -1,71 +1,88 @@
...
-Possimus saepe veritatis sint nobis et quam eos. Architecto consequatur odit perferendis fuga eveniet possimus rerum cumque. Ea deleniti voluptatum deserunt voluptatibus ut non iste. Provident nam asperiores vel laboriosam omnis ducimus enim nesciunt quaerat. Minus tempora cupiditate est quod.
-
-### Reprehenderit magni
-
-Sit commodi iste iure molestias qui amet voluptatem sed quaerat. Nostrum aut pariatur. Sint ipsa praesentium dolor error cumque velit tenetur quaerat exercitationem. Consequatur et cum atque mollitia qui quia necessitatibus.
-
-Voluptas beatae omnis omnis voluptas. Cum architecto ab sit ad eaque quas quia distinctio. Molestiae aperiam qui quis deleniti soluta quia qui. Dolores nostrum blanditiis libero optio id. Mollitia ad et asperiores quas saepe alias.
+---
+title: CAOS installation guide
+description: A guide on installing the required software to contribute to CAOS
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
description: A guide for installing the required software to contribute to CAOS

```

File: src/pages/docs/installation.md [hunk 8]
```diff
@@ -1,71 +1,88 @@
...
+---
+
+A guide on installing the required software to contribute to CAOS
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
A guide for installing the required software to contribute to CAOS

```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by tiago-bacelar — CHANGES_REQUESTED** [review]
Make the script to work for ZSH. Currently ASDF is only installed for Bash.

File: src/pages/docs/installation.md [hunk 9]
```diff
@@ -1,71 +1,91 @@
...
+### Docker
 
-### Voluptatem quas possimus
+Docker is a tool that allows us to run our applications in (containers)[https://en.wikipedia.org/wiki/Containerization_(computing)]. Again, this allows us to have a development environment that is really close to the production one, reducing the risk of hiding / creating bugs that would only happen on one environment.
 
-Sit commodi iste iure molestias qui amet voluptatem sed quaerat. Nostrum aut pariatur. Sint ipsa praesentium dolor error cumque velit tenetur quaerat exercitationem. Consequatur et cum atque mollitia qui quia necessitatibus.
+To install Docker do the following:
 
-Possimus saepe veritatis sint nobis et quam eos. Architecto consequatur odit perferendis fuga eveniet possimus rerum cumque. Ea deleniti voluptatum deserunt voluptatibus ut non iste. Provident nam asperiores vel laboriosam omnis ducimus enim nesciunt quaerat. Minus tempora cupiditate est quod.
+| OS | Command |
```
Comment (tiago-bacelar) [inline src/pages/docs/installation.md]: This table isn't formatted correctly
image

**Review by tiago-bacelar — CHANGES_REQUESTED** [review]

File: src/pages/docs/installation.md [hunk 10]
```diff
@@ -1,71 +1,92 @@
 ---
-title: Installation
-description: Quidem magni aut exercitationem maxime rerum eos.
+title: CAOS installation guide
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
title: CeSIUM installation guide
```

File: src/pages/docs/installation.md [hunk 11]
```diff
@@ -1,71 +1,92 @@
...
+description: A guide to installing the required software to contribute to CAOS
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
description: A guide to installing the required software to contribute to CeSIUM
```

File: src/pages/docs/installation.md [hunk 12]
```diff
@@ -1,71 +1,92 @@
...
 ---
 
-Quasi sapiente voluptates aut minima non doloribus similique quisquam. In quo expedita ipsum nostrum corrupti incidunt. Et aut eligendi ea perferendis.
+A guide to installing the required software to contribute to CAOS
```
Comment (MarioRodrigues10) [inline src/pages/docs/installation.md]:
```suggestion
A guide to installing the required software to contribute to CeSIUM
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by MarioRodrigues10 — APPROVED** [review]

**Review by tiago-bacelar — APPROVED** [review]

**Pull request merged**